styles.css

```diff
@@ -13,11 +13,12 @@ html {
 body {
   max-width: max-content;
   margin: auto;
+  height: 100%;
+  border: 1px solid black;
 }
```

Comment:
[Optional] Kindly remove the black border in the body of the page as this does not appear in the design template.

Your design:
![image](https://user-images.githubusercontent.com/20554583/192437806-ff48d38a-bf43-4325-ba80-4aa03922b890.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/192437889-05f7a962-d454-48b4-b75a-3b91a5d0aebf.png)